Tests/FluentAssertions.Shared.Specs/GenericDictionaryAssertionSpecs.cs

```diff
@@ -917,6 +917,17 @@ public void Should_succeed_when_asserting_dictionary_does_not_contain_a_key_that
         }
 
         [TestMethod]
+        public void Should_succeed_when_asserting_dictionary_does_not_contain_multiple_keys_from_the_dictionary()
```

Comment:
I actually copied both the structure and name from the existing test for Contain. That is why I thought it was correct. I simply changed _contains_ to _does_not_contain_ in the name.